examples_test.go

```diff
@@ -150,42 +150,52 @@ func testExample(testingT *testing.T, name string, conf goose.Config) {
 			t.GoldFile())
 	}
 	if actual != expected {
+		t.PutActual(actual)
```

Comment:
I noticed that this line was previously after the `assert.FailNowf`, and it wouldn't actually end up producing output.